### ISSUE TITLE
Open file dialog doesn't show and tab strings are not visible

### ISSUE THREAD

**Issue by ms1331** [agourlay/ruxguitar#2]
Thanks for the cool project! I am trying to get it to run on Archlinux with wayland, but am having some issues. The file open dialog doesn't show unless I modify the Cargo.toml and replace:

```
rfd = "0.14.1"
```

with 
```
rfd = { version = "0.14.1", default-features = false, features = ["gtk3"] }
```
The other issue is when I load one of the example tabs I am getting the following output

![bla](https://github.com/user-attachments/assets/cc29e351-1801-4c2c-8110-55160d3ba62b)

Last thing is I had to install the pipewire-alsa package to get it to start, but that's platform dependentan

Other than that it looks pretty nice for 0.1.0 release!

**Comment by agourlay** [comment]
Thanks for the investigation!

Are you seeing the following log line on the vanilla binary?

```
WARN ruxguitar::ui::application] Failed to read GP file: dialog window closed without selecting a file
```

**Comment by ms1331** [comment]
Yes, I am seeing that exact warning.

**Comment by agourlay** [comment]
For the display issue, maybe you could try disabling the `antialiasing` in https://github.com/agourlay/ruxguitar/blob/master/src/ui/application.rs#L113

**Comment by ms1331** [comment]
Changing the antialiasing to:

```
        .antialiasing(false)
```

fixes the strings display... Thanks!

**Comment by agourlay** [comment]
Lucky shot :)

I will setup a flag in the CLI to disable antialiasing on demand then.

**Comment by agourlay** [comment]
Regarding the file picker issue, is it possible for you to install the `XDG Desktop portal`?

**Comment by ms1331** [comment]
So installing the xdg-desktop-portal-gtk fixed the file picker.

**Comment by agourlay** [comment]
Fixed in [0.1.1](https://github.com/agourlay/ruxguitar/releases/tag/v0.1.1) which:
- introduced `no-antialiasing` options
- FAQ with the XDG desktop portal requirement on Linux
